sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/subscriptions/SubscriptionsDbConnector.groovy

```diff
@@ -105,22 +105,24 @@ class SubscriptionsDbConnector implements SubscriptionDataSource, SubscribedProj
     }
     
     private int getSubscriberId(Connection connection, Subscriber subscriber) {
-          int subscriberId = fetchExistingSubscriberId(subscriber)
-          if(subscriberId <= 0) {
-            String query = "INSERT INTO subscriber (first_name, last_name, email) VALUES(?, ?, ?)"
-            
+        int subscriberId = fetchExistingSubscriberId(subscriber)
+        if(subscriberId <= 0) {
+            String query = "INSERT INTO person (user_id, first_name, last_name, title, email) VALUES(?, ?, ?, ?, ?)"
```

Comment:
We must not do this, this violates the SRP of this application. We have to find another way, to assure that the logged in person has an entry in the Person table.